index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable max-classes-per-file */
+const form = document.querySelector('#form');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const cardsContainer = document.querySelector('#cards');
+
+const warningLabel = document.querySelector('.warningLabel');
+
+class Storage {
+  static addTodStorage(arrBooks) {
+    const storage = localStorage.setItem('book', JSON.stringify(arrBooks));
+    return storage;
+  }
+
+  static getStorage() {
+    const storage = localStorage.getItem('book') === null
+      ? []
+      : JSON.parse(localStorage.getItem('book'));
+    return storage;
+  }
+}
+
+// array
+let arrBooks = Storage.getStorage();
+const id = 1;
+
+// object books
+class Books {
+  constructor(id = 0, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+}
+// display books
+class Card {
+  static displayBook() {
+    const displayBook = arrBooks.map(
+      (item) => `
+        <div class="card border-0">
+              <h5 class="card-title">${item.title}</h5>
+              <p class="card-text">${item.author}</p>
+              <button class="btn btn-secondary remove w-25" data-id="${item.id}">remove</button>
+              <hr/>
+            </div>`,
+    );
+    cardsContainer.innerHTML = displayBook.join(' ');
+  }
+
+  static clearInputs() {
+    bookTitle.value = '';
+    bookAuthor.value = '';
+  }
+
+  static removeBook() {
+    cardsContainer.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove')) {
+        e.target.parentElement.remove();
+      }
+      const bookId = e.target.dataset.id;
+      Card.removeBookFromArr(bookId);
+    });
+  }
+
+  static removeBookFromArr(id) {
+    arrBooks = arrBooks.filter((item) => item.id !== +id);
+    Storage.addTodStorage(arrBooks);
+  }
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Data should be `preserved` in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), `reload` the page manually to check this after adding a book and removing a book. When I `remove` one of the many added books and `reload` the page, the rest of the added `books` are also removed from the `array`. Please fix it 🙏🏼 

![Bug](https://user-images.githubusercontent.com/63915024/211380784-34199ce2-9399-4cda-89e3-0fa757eac18d.gif)

Reply:
well resolved! ✔️